addons/bindings.md

```diff
@@ -6,9 +6,11 @@ layout: documentation
 
 # List of Available Bindings
 
-{% assign bindings = site.data.bindings | sort: 'label.toLowerCase()' %}
+{% assign bindings = "" | split: "|" %}
```

Comment:
what is the "split" good for here? To make it an array? But doesn't it then contain an empty string as a first element?